demo/src/index.js

```diff
@@ -21,16 +22,22 @@ class Demo extends Component {
                 className="ReactWebChat"
             >
                 <ReactWebChatComponent
-                    url="https://dev.feersum.io/chat_sockjs"
+                    // url="https://dev.feersum.io/chat_sockjs"
+
+                    url="http://localhost:8000"
                     typingStatus={{
                         active: true,
                         baseDelay: 1000,
                         variance: 500
                     }}
                     network={{
-                        channel_id: '3998e8a9-b329-4de9-a03c-040cc0348e42'
+                        // channel_id: '3998e8a9-b329-4de9-a03c-040cc0348e42'
+
+                        channel_id: '6567c91f-1ece-4289-ba7a-1a2aa34504c9',
+                        transportServerUrl: 'http://127.0.0.1:8050'
```

Comment:
@mfundontini Once we deploy Feersum will you please let me know the QA Transport server URL